services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+
+from datetime import datetime, date
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: date,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+def get_movies_sessions(
+    session_date: str = None
+) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        date_obj = datetime.strptime(session_date, '%Y-%m-%d')
+        return queryset.filter(session_date=date_obj)
+    else:
+        return queryset
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: None,

```

Comment:
The parameter `show_time` in `update_movie_session` is incorrectly set to `None`. It should be `show_time: datetime = None` to allow optional updating of the show time.